cypress/integration/payment/create_manual_payment.js

```diff
@@ -0,0 +1,185 @@
+/// <reference types="Cypress" />
+
+import {BPartner} from '../../support/utils/bpartner';
+import {SalesInvoice, SalesInvoiceLine} from '../../support/utils/sales_invoice';
+import {DiscountSchema} from "../../support/utils/discountschema";
+import {Bank} from "../../support/utils/bank";
+import {Builder} from "../../support/utils/builder";
+
+describe('Create a manual Payment for a Sales Invoice', function () {
+  const timestamp = new Date().getTime();
+  // const timestamp = 'latest timestamp'; // to use this just comment the `before` function
+
+
+  const salesInvoiceTargetDocumentType = 'Sales Invoice';
+  let salesInvoiceNumber;
+  let salesInvoiceTotalAmount = 0;
+  let salesInvoiceID = 0;
+
+  const paymentDocumentType = 'Zahlungseingang';
+  let paymentTotalAmount = 0; // may be different from salesInvoiceTotalAmount because there could be some discounts
+
+
+  // data for "before" section
+  // priceList
+  const priceSystemName = `PriceSystem ${timestamp}`;
+  const priceListName = `PriceList ${timestamp}`;
+  const priceListVersionName = `PriceListVersion ${timestamp}`;
+
+  // product
+  const productCategoryName = `ProductCategory ${timestamp}`;
+  const productCategoryValue = productCategoryName;
+  const productName = `Product ${timestamp}`;
+  const productValue = productName;
+
+  // BPartner
+  const discountSchemaName = `DiscountSchema ${timestamp}`;
+  const bPartnerName = `bPartner ${timestamp}`;
+
+
+  before(function () {
+    Builder.createBasicPriceEntities(priceSystemName, priceListVersionName, priceListName);
+
+    Builder.createBasicProductEntities(productCategoryName, productCategoryValue, priceListName, productName, productValue);
+
+    cy.fixture('discount/discountschema.json').then(discountSchemaJson => {
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+    cy.fixture('finance/bank.json').then(productJson => {
+      Object.assign(new Bank(), productJson)
+        .apply();
+    });
+    cy.fixture('sales/simple_customer.json').then(customerJson => {
+      Object.assign(new BPartner(), customerJson)
+        .setName(bPartnerName)
+        .setCustomerDiscountSchema(discountSchemaName)
+        .apply();
+    });
+
+    cy.readAllNotifications();
+  });
+
+
+  it('Creates a Sales Invoice', function () {
+    new SalesInvoice(bPartnerName, salesInvoiceTargetDocumentType)
+      .addLine(
+        new SalesInvoiceLine()
+          .setProduct(productName)
+          .setQuantity(20)
+        // todo @dh: how to add packing item
+        // .setPackingItem('IFCO 6410 x 10 Stk')
+        // .setTuQuantity(2)
+      )
+      // .addLine(
+      // todo @dh: how to add this line which depends on the packing item?
+      //   new SalesInvoiceLine()
+      //     .setProduct('IFCO 6410_P001512')
+      //     .setQuantity(2)
+      // )
+      .setPriceList(priceListName)
+      .setDocumentAction('Complete')
```

Comment:
this will fail when running with a de_DE user.
what should actually work is this

```js
    cy.fixture('misc/misc_dictionary.json').then(miscDictionary => {
      cy.processDocument(
        getLanguageSpecific(miscDictionary, 'docActionComplete'),
        getLanguageSpecific(miscDictionary, 'docStatusCompleted')
      );
    });
```

Reply:
fixed